2-mandatory/2-financial-times.js

```diff
@@ -23,19 +40,36 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+    const regex = /[0-9]/;
+    let result = [];
+    for (let articleTitle of allArticleTitles) {
```

Comment:
Looks great!
As an extra challenge, can you think of another implementation which doesn't require nested for loops?